pkg/test/resourcefixture/testdata/basic/netapp/v1alpha1/netappbackupvault/netappbackupvault-basic/_http.log

```diff
@@ -29,7 +29,7 @@ Content-Type: application/json
 Grpc-Metadata-Content-Type: application/grpc
 
 {
-  "done": false,
+  "done": true,
```

Comment:
Could you merge the last two commits and re-upload? It's quite difficult to see the diff between the real and mock.

Reply:
done!

Reply:
Btw you can view the diff by looking at the commit prior: https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/4141/commits/6ca97c177142ece82bc5850989a5313dab59f5ca